src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -345,6 +345,14 @@ public function getParameterOption(string|array $values, string|bool|int|float|a
         return $default;
     }
 
+    /**
+     * Returns un-parsed and not validated tokens.
```

Comment:
Please add `@return list<string>` to provide the precise type of what is returned in this array.

Reply:
Good idea! rebased + fixed + pushed